projects/storefrontlib/src/cms-components/myaccount/order/amend-order/amend-order.service.ts

```diff
@@ -14,6 +14,16 @@ function ValidateQuantity(control: FormControl) {
   return q > 0 ? null : { required: true };
 }
 
+function ValidateQuantityToCancel(control: FormControl) {
+  let q = 0;
```

Comment:
```suggestion
  const quantity = Object.entries(control.value.entries).reduce((acc, val) => acc + val, 0);
  
  return quantity > 0 ? null : { cxNoSelectedItemToCancel: true };
```

Reply:
Contents of this whole function can be replaced with `reduce()`. Please check if this solution is covering the problem.

Reply:
Good idea, thanks. Works fine. Just had to add simple null-check, as object seems to happen to be null.